templates/snap-details.html

```diff
@@ -97,14 +97,35 @@ <h1 class="p-heading--two p-snap-heading__name">{{ snap_title }}</h1>
         </table>
       </div>
     </div>
-    {% if countries %}
-      <div class="row js-snap-map-holder" style="margin-top: 2rem;">
-        <div class="col-12">
+    <div class="row" style="margin-top: 2rem;">
```

Comment:
It's a bit weird to put margin in inline styles here.

I guess the easiest way to get additional spacing would be to wrap charts into `p-strip is-shallow`, or have some custom class that adds necessary spacing.